src/app/room/components/room.component.ts

```diff
@@ -52,13 +52,11 @@ export class RoomComponent implements OnInit {
     this.loadingStatus.isLoading = true;
     this.currentPlayer$.pipe(take(1))
       .pipe(
-        map(player => this.firebasePlayerService.remove(player.playerId)),
+        map(player => merge(this.playerInRoomService.remove(player.playerInRoomId), this.firebasePlayerService.remove(player.playerId))),
```

Comment:
We decided to not remove the player (in case of having an account) and to still link it to `playerId `in the statistics collections

Reply:
Hm yes I remember now. But does that really make sense as long as there is no login mechanism. We don't really have any concept of identity right? If a player clears local storage or leaves the room his ID is lost forever, why then connect the statistics. 
Maybe it would make more sense to postpone this connection until we have a way to identify a user really.
Alternatively, if we keep this player now, we should remove all room-related information, meaning roomId and choices, from it, and only keep ID and name right?